packages/trader/src/Modules/SmartChart/Components/recent-trade-info.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
```

Comment:
You just separate this component, correct? I mean we don't need sanity, right?

Reply:
Maybe `localize` can be replaced?

Reply:
yeah separated it to be make it easy to write test